### PR TITLE
Add section to README about how to configure for testing

### PR DESCRIPTION
**Pull request by jeremywadsack**
Addresses request in #41.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,6 +303,67 @@ retriable_with_context(:api) do
 end
 ```
 
+## In Tests
```
Comment (apurvis) [inline README.md]: this all looks great but maybe you can retitle this?  when i first red this i assumed it was about running tests for the gem... so maybe something like "Short Circuiting Retriable While Testing Your App"

i might also add a note about what exactly the problem is (namely, that it often takes a long time to retry blocks that fail)

Comment (jeremywadsack) [reply README.md]: @apurvis 425f2d9. Let me know if that is clearer. I can squash commits if this looks good.

**Comment by apurvis** [comment]
lgtm (though only @kamui can merge)

File: README.md [hunk 2]
```diff
@@ -303,6 +303,69 @@ retriable_with_context(:api) do
 end
 ```
 
+## Short Circuiting Retriable While Testing Your App
+
+When you are running tests for your app, it often takes a long time to retry blocks that fail. This is because Retriable will default to 3 tries with exponential backoff. Ideally, your tests will run as quickly as possible.
```
Comment (apurvis) [inline README.md]: just to nitpick some grammar: you should get rid of both commas here.

otherwise thanks; this will definitely help someone else

**Comment by jeremywadsack** [comment]
@apurvis Fixed the commas. I always overuse them.

Squashed commits. Should be ready to merge.

**Pull request merged**